### ISSUE TITLE
[BUG] URGENT something completely breaks all camera controls

### ISSUE THREAD

**Issue by xoxifor** [bloxstraplabs/bloxstrap#3136]
### Acknowledgement of preliminary instructions

- [X] I have read the preliminary instructions, and I am certain that my problem has not already been addressed.
- [X] I have thoroughly looked through the available Wiki articles and could not find a solution to my problem.
- [X] I am using the latest version of Bloxstrap.
- [ ] I did not answer truthfully to all the above checkboxes.

### What problem did you encounter?

so basically when right clicking normally the mouse kind of gets locked in place same with first person and shift lock however something COMEPLETLY breaks that functionality making certain games UNPLAYABLE i.e deepwoken iykyk 

### Bloxstrap Log

N/A

**Comment by xoxifor** [comment]
not a blox strap issue
